src/components/RequestPanel/RequestStepItem.js

```diff
@@ -146,7 +145,7 @@ function TransactionStepItem({ hash, status, stepNumber, transaction }) {
             ${textStyle('title4')}
           `}
         >
-          {numberToOrdinal(stepNumber)} transaction
+          {capitalize(numberToOrdinal(stepNumber))} request
```

Comment:
Similar to above, I think this is a bit weird and we might want to re-think the copy.